Source/Editor/Viewport/PrefabWindowViewport.cs

```diff
@@ -137,6 +208,12 @@ public PrefabWindowViewport(PrefabWindow window)
             _uiRoot = new PrefabUIEditorRoot(this);
             _uiRoot.IndexInParent = 0; // Move viewport down below other widgets in the viewport
             _uiParentLink = _uiRoot.UIRoot;
+            
+            var container = new ViewportWidgetsContainer(ViewportWidgetLocation.UpperLeft);
+            container.Parent = this;
+            _uiModeButton = new ViewportWidgetButton("UI Mode", SpriteHandle.Invalid, null, true);
```

Comment:
I think we should use exiting `View` button and add a checkbox for `UI Mode` (via `ViewWidgetShowMenu`).

Reply:
Done